.buildkite/scripts/steps/cloud/deploy.json

```diff
@@ -213,7 +213,10 @@
           ],
           "kibana": {
             "version": null,
-            "docker_image": null
+            "docker_image": null,
+            "user_settings_json": {
+              "xpack.securitySolution.enableExperimental": ["extendedRuleExecutionLoggingEnabled"]
```

Comment:
I know we're testing things here, just adding a note so we remember to revert i.

Reply:
@jbudz I reverted changes in this file, could you please check and approve the PR?